src/layouts/Docs.tsx

```diff
@@ -241,7 +249,7 @@ export const DocsLayout = ({
           <Translation id="page-developers-docs:banner-page-incomplete" />
         </BannerNotification>
       )}
-      <ContentContainer>
+      <ContentContainer dir={contentNotTranslated ? "ltr" : "unset"}>
```

Comment:
This is a primary situation where `dir` should be used to override existing logic. 👍🏼